content/en/real_user_monitoring/ios/advanced_configuration.md

```diff
@@ -127,7 +138,7 @@ Find more details and available options in the [`DDRUMMonitor` class][9].
 
 ### Custom Resources
 
-In addition to [tracking resources automatically](#automatically-track-network-requests), you can also track specific custom resources such as network requests or third-party provider APIs. Use the following methods on `Global.rum` to manually collect RUM resources:
+In addition to [tracking resources automatically](#automatically-track-network-requests), you can also track specific custom resources such as network requests or third-party provider APIs. Use the following methods on `RUMMonitor.shared()` to manually collect RUM resources:
 
 - `.startResourceLoading(resourceKey:request:)`
 - `.stopResourceLoading(resourceKey:response:)`
```

Comment:
Here and more in other snippets:
> resource tracking APIs are renamed - we no longer use word "Loading":
> - startResourceLoading() → startResource()
> - stopResourceLoading() → stopResource()
> - stopResourceLoadingWithError() → stopResourceWithError()

Ref. _breaking changes_ from https://github.com/DataDog/dd-sdk-ios/pull/1310